helm/values.yaml

```diff
@@ -78,6 +78,11 @@ kubernetes:
     type: ClusterIP
     port: 443
 
+  # configure Connaisseur serviceaccount (so you can do IRSA, for example)
+  serviceaccount:
```

Comment:
This looks like something that we'd previously have commented out in the default helm because it's not really neccessary and can be ignored by most people.

Would be fine with approving, if you want it this way though